templates/gatekeeper.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: kubeaddons.mesosphere.io/v1alpha1
+kind: Addon
+metadata:
+  name: gatekeeper
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: gatekeeper
+  annotations:
+    appversion.kubeaddons.mesosphere.io/gatekeeper: "3.0.4-beta.1"
+    docs.kubeaddons.mesosphere.io/gatekeeper: "https://github.com/open-policy-agent/gatekeeper/blob/master/README.md"
+    values.chart.helm.kubeaddons.mesosphere.io/gatekeeper: "https://raw.githubusercontent.com/mesosphere/charts/master/staging/gatekeeper/values.yaml"
+spec:
+  kubernetes:
+    minSupportedVersion: v1.15.0
+  cloudProvider:
```

Comment:
I'd add a requires label here as in other addons to `cert-manager`. We could run the gatekeeper with cert-manager and this addon is required so we are safe adding it in my opinion.

Reply:
Makes sense. Added `cert-manager` dependency.